client/src/test/java/com/paypal/selion/TestServerListener.java

```diff
@@ -25,7 +28,7 @@ public void onStart(ISuite suite) {
         try {
             TestServerUtils.startServer();
         } catch (Exception e) { // NOSONAR
-            e.printStackTrace();
+            SeLionLogger.getLogger().log(Level.WARNING, e.getMessage());
```

Comment:
I don't see the need to put this in the SeLion log. It is Test code. We should just print a message that says something like `Unable to start test server` to `System.err`